content/contributing/project-information/release-process.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Release Process"
+insert_anchor_links = "right"
+[extra]
+weight = 3
++++
+
+Bevy uses three-months-long development cycles, delimited by a weeks-long rolling release process.
+
+This means that bevy releases a new version roughly once every three to four months. In an effort to avoid crunch and encourage sustainable development, we do not pick specific dates or deadlines.
+
+## Development Cycles
+
+During a three-month development cycle, maintainers collect important PRs and issues into a GitHub Milestone. Issues included in the milestone are given priority, and the community generally makes a best-effort attempt to merge as many of these as possible over the duration of the cycle.
+
+In the lead-up to a release (generally about about two and a half months after the previous) we begin to aggressively purge the milestone of things that seem unlikely to be ready in time. The main things that tend to remain are:
+
+- Quick and easy features under active development.
+- Fixes for important regressions or crashes.
+- Large or impressive features that only need minor polish.
+
+When the milestone is complete, we publish a release candidate (ex. `0.14.0-rc.1`) and begin the release process.
+
+## Release Candidates
+
+In the run-up to a release, Bevy publishes a new release candidate about once a week. Publishing release candidates allows users to identify bugs and regressions, gives ecosystem crates time to update so they can be ready closer to release day, and provides bevy contributors extra time to write release notes and migration guides.
```

Comment:
I think this last bit isn't worded strongly enough. I frankly think it deserves its own paragraph.

Once again we are neglecting to acknowledge the monumental effort required for release prep (the blog post, etc) and how much time that takes!

The way I see it, the benefit of the RC period is just as much (if not more) about giving maintainers + the community time for release prep work (such as writing the blog post), as it is about code quality.

I think it should be more clearly communicated.

Release prep requires a monumental amount of effort and I believe that should be acknowledged. Y'all need the time to be able to do it stress-free and crunch-free.

Previous releases have been very close to driving people to burnout because this isn't more explicitly addressed.

There is a lot of focus on "we don't set specific deadlines to avoid crunch for devs" (in the previous section about the dev cycle), but not enough on avoiding crunch for the people involved in release prep!

I have been advocating for a clear official "release prep period" after the 3-month dev cycle primarily for this reason!

Reply:
I want to tackle this in a dedicated follow-up :)